tests/unit/worker/test_build_dataset_worker.py

```diff
@@ -261,9 +261,7 @@ def test_import(requests_mock: RequestsMocker):
         assert request_matches(
             requests_made[index], EXPECTED_REQUESTS_IMPORT[index]
         )
-    assert len(
-        get_file_list_from_dir(Path(INPUT_DIR_ARCHIVE) / f'{DATASET_NAME}')
-    ) == 2
+    assert (Path(INPUT_DIR_ARCHIVE) / f'{DATASET_NAME}').exists()
```

Comment:
Shouldn't this evaluate to false since this job succeeds?

Reply:
Yes, it should. I'll look into it 👍

Reply:
After a closer look the answer is: the assert in the test is correct. The archived dataset is deleted at the end of the manager phase and this test is only considering the worker phase. So all good.

Reply:
Ah! Of course! That makes sense 👍🏻